pyfstat/helper_functions.py

```diff
@@ -478,23 +477,21 @@ def get_doppler_params_output_format(keys):
         "tp",
         "argp",
     ]
-
-    for k in keys:
-        if k in doppler_keys:
-            fmt += [CFSv2_fmt]
+    fmt = {k: CFSv2_fmt for k in keys if k in doppler_keys}
```

Comment:
Reminder to check where is this function used

Reply:
```
grep -n get_doppler_params_output_format pyfstat/*.py
pyfstat/grid_based_searches.py:340:        fmt = helper_functions.get_doppler_params_output_format(self.output_keys)
pyfstat/grid_based_searches.py:845:        fmt = helper_functions.get_doppler_params_output_format(self.output_keys)
pyfstat/grid_based_searches.py:968:        fmt = helper_functions.get_doppler_params_output_format(self.output_keys)
pyfstat/helper_functions.py:464:def get_doppler_params_output_format(keys):
pyfstat/mcmc_based_searches.py:1722:        fmt = helper_functions.get_doppler_params_output_format(self.theta_keys)
pyfstat/mcmc_based_searches.py:2485:        fmt = helper_functions.get_doppler_params_output_format(
pyfstat/mcmc_based_searches.py:2488:        fmt = helper_functions.get_doppler_params_output_format(self.theta_keys)
pyfstat/mcmc_based_searches.py:3388:        fmt = helper_functions.get_doppler_params_output_format(self.theta_keys)
```

All use it in `dict` fashion, but `pyfstat/mcmc_based_searches.py:2485` + `pyfstat/mcmc_based_searches.py:2488` makes no sense, let me see what and why I did there.

Reply:
Duplication resolved, but it was helpful in noticing and fixing a sorting issue with the final output formats list.